.copyright-ignore

```diff
@@ -0,0 +1,45 @@
+# Copyright 2023, The OpenTelemetry Authors
+# SPDX-License-Identifier: Apache-2.0
+
+# Ignore the following directories
+
+./.git/*
+./.github/*
+./third_party/*
```

Comment:
nit: under third_party, there is BUILD file which is still for this repo and needs check. Could we specify some files are checked explicitly even matching the ignore pattern? But it is Ok to not have this.

Reply:
Fixed by using this instead:

```
./third_party/benchmark/*
./third_party/boost/*
./third_party/googletest/*
./third_party/ms-gsl/*
./third_party/nlohmann-json/*
./third_party/opentelemetry-proto/*
./third_party/prometheus-cpp/*
```